### PR TITLE
ignore intellij project folder. minor cleanup.

### PR DESCRIPTION
**Pull request by lookfirst**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,9 +1,8 @@
 node_modules
 sandbox
-testlibs/jspm_packages
-testlibs/build.js
-testlibs/build.js.map
+.idea
 .DS_Store
-testlibs/.DS_Store
```
Comment (guybedford) [inline .gitignore]: Spare a thought to a poor mac user.

Comment (lookfirst) [inline .gitignore]: I'm a mac user too! .gitignore is recursive. Try it. =)

Comment (lookfirst) [inline .gitignore]:
```
jon:jspm-cli jon$ cat .gitignore
node_modules
sandbox
.idea
.DS_Store
npm-debug.log
testlibs/jspm_packages
testlibs/build.js*
test/outputs
jon:jspm-cli jon$ touch testlibs/.DS_Store
jon:jspm-cli jon$ git st
On branch master
Your branch is up-to-date with 'origin/master'.
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	modified:   .gitignore
```

Comment (guybedford) [inline .gitignore]: Ha, recursive in dot files, interesting!

**Pull request merged**